cmd/state-svc/main.go

```diff
@@ -122,16 +122,25 @@ func run(cfg *config.Instance) error {
 	)
 
 	var foregroundArgText string
+	var startup bool
 
 	cmd.AddChildren(
 		captain.NewCommand(
 			cmdStart,
 			"",
 			"Start the ActiveState Service (Background)",
-			p, nil, nil,
+			p,
+			[]*captain.Flag{
+				{Name: "startup", Value: &startup}, // differentiate between autostart and cli invocation
```

Comment:
I'm not sure if we have curated the help output for the service, but I suggest setting Hidden to true.

Reply:
I suggest renaming "startup" to autostart.